### ISSUE TITLE
Questions about siamese network details

### ISSUE THREAD

**Issue by Liudzz** [HarshSulakhe/siamesenetworks-pytorch#1]
Hi , thanks for your generous sharing ! 

I'm interested in siamese network and contrastive loss . I was so happy when I searched your code,and I have learn a lot in your code.But some problems are still confusing me .It is great if you can help me.
 
The problem is parameters of _margin_ and _threshold_. I don't change your parameters in my code, and I use _evaluate_pair_ in my train process .I find that  _neg_num/neg_all_ is always 1 while _pos_num/pos_all_ is always 0 . So I think maybe  _threshold_ is too small ? I also find that in your code , **N acc**  is 1 but **p acc** has not change too much .How do you deal with it and how do you finetune parameters?

It's kind of you if you can respond. : )
